db/seeds.rb

```diff
@@ -18,15 +18,15 @@
 
 5.times do
   user = User.new(username: Faker::Internet.username, email: Faker::Internet.email, password: "123456", password_confirmation: "123456", bio: Faker::Hacker.say_something_smart)
-  file = URI.open("https://www.rd.com/wp-content/uploads/2017/09/01-shutterstock_476340928-Irina-Bg-1024x683.jpg")
+  file = URI.open("https://loremflickr.com/1000/1000")
```

Comment:
change loremflickr to specifically search for a person/portrait, or use unsplash service